scripts/reports/cloc-repository.sh

```diff
@@ -28,20 +26,44 @@ function main() {
   enrich-report
 }
 
+# Create the report.
 function create-report() {
```

Comment:
Could this be renamed to be create-lines-of-code-report?

Reply:
done, thanks

Reply:
I changed the file name for clarity.